src/gui/SetupTotpDialog.cpp

```diff
@@ -0,0 +1,74 @@
+#include "SetupTotpDialog.h"
```

Comment:
@weslly the copyright header is missing from this file (and some other new files in this PR)